filament/backend/test/PlatformRunner.h

```diff
@@ -34,6 +34,13 @@ enum class Backend : uint8_t {
     NOOP = 5,
 };
 
+
+enum class OperatingSystem: uint8_t {
+    OTHER = 1,
+    LINUX = 2,
+    MACOS = 3,
+};
```

Comment:
Should we add entries for Windows/iOS?

Reply:
I wasn't sure, given that the tests don't support windows in the first place and iOS is just MacOS but with the isMobile flag set. Similar to how android is essentially linux with the isMobile flag.
Should I rename MACOS to APPLE? Or make this munge together isMobile and the OS field?
Maybe I smush together all of the options into an SkippedEnvironment struct with optional fields for OS, isMobile, and Backend, and skips if every non-optional field matches.